governance.md

```diff
@@ -323,3 +323,19 @@ Existing **Tier 2** Partners:
 
 -   Bloomberg (Jason Grout, Sylvain Corlay)
 -   Google (Kester Tong, Kayur Patel, Mark Sandler)
+
+
+Changing the Governance
+=======================
+
+Changes to the governance documents are proposed via a GitHub pull
+request to The Project's governance documents GitHub repository at
+[https://github.com/jupyter/governance](https://github.com/jupyter/governance).
+After a period of public comment and review, a Steering Council Member
+proposes to the Steering Council that the pull request be merged
+(accepting the proposed changes) or proposes that the pull request be
+closed without merging (rejecting the proposed changes).  The Member
+should list the commit range proposed for acceptance or rejection and
+briefly summarize the pull request.  A majority vote from the Steering
+Council is needed to carry out the action.  The BDFL has authority to
```

Comment:
I think we do need language here that expresses our desire and intention for the open, community based discussion to reach consensus before a vote is proposed.